timemachine/fe/free_energy.py

```diff
@@ -59,22 +59,17 @@ class HREXParams:
     n_frames_bisection: int
         Number of frames to sample using MD during the initial bisection phase used to determine lambda spacing
 
-    n_frames_per_iter: int
-        Number of frames to sample using MD per HREX iteration.
-
     max_delta_states: int or None
         If given, number of neighbor states on either side of a given replica's initial state for which to compute
         potentials. This determines the maximum number of states that a replica can move from its initial state during
         a single HREX iteration. Otherwise, compute potentials for all (replica, state) pairs.
     """
 
     n_frames_bisection: int = 100
-    n_frames_per_iter: int = 1
```

Comment:
Is there a backwards compatibility issue of loading old pkl'd HREXParams with this removed?

Reply:
Yes, I think so, but also seemed confusing to leave it... I'm not sure what our expectations should be re backwards compatibility here; happy to defer to others.

Reply:
Maybe leave it and just assert it equals one in the check below?

Reply:
We should know which versions of timemachine we have per pickle, which as long as we know that we should be able to debug still

Reply:
> Maybe leave it and just assert it equals one in the check below?

That works for me

Reply:
Reverted with assertion in https://github.com/proteneer/timemachine/pull/1325/commits/a49818818b3109b4cbe4c18cce94d08b91dde5db